Gemfile

```diff
@@ -15,3 +15,7 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+group :test do
+  gem "timecop"
+end
```

Comment:
It too bad we can't just use https://github.com/rails/rails/blob/4-1-stable/activesupport/lib/active_support/testing/time_helpers.rb  since they aren't in Rails 4.0